web/src/utilities/utilities.js

```diff
@@ -55,6 +55,22 @@ export function getApplicationType(watch) {
   }
 }
 
+export function getReadableCollectorName(name) {
+  const namesObj = {
+    "cluster-info": "Gathering basic information about the cluster",
+    "cluster-resources": "Gathering available resources in cluster",
+    "mysql": "Gathering information about MySQL",
+    "postgres": "Gathering information about PostgreSQL",
+    "redis": "Gathering information about Redis"
+  }
+  const statusToReturn = namesObj[name];
+  if (statusToReturn) {
+    return statusToReturn
+  } else {
+    return `Gathering details about the cluster`;
```

Comment:
why `? can this be "Gathering details about the cluster"?

Reply:
I'm not sure I follow.

Reply:
does it need to be as backtick? or it can be inside quotation mark?

Reply:
Oh I see. No can be quotation marks.